pkgs/top-level/all-packages.nix

```
@@ -5065,6 +5065,9 @@ with pkgs;
     inherit (emacs.pkgs.melpaStablePackages) irony;
   };
 
+  iroh-relay = callPackage ../tools/networking/iroh/iroh-relay.nix { };
+  iroh-dns-server = callPackage ../tools/networking/iroh/iroh-dns-server.nix { };
```

This needs to follow the code pattern I linked above, or the `nixpkgs-vet` check will not pass.

```suggestion
  inherit ({
    iroh-relay = callPackage ../tools/networking/iroh/iroh-relay.nix { };
    iroh-dns-server = callPackage ../tools/networking/iroh/iroh-dns-server.nix { };
  })
  iroh-relay
  iroh-dns-server;
```

---

What? I never seen this pattern... Is there any rationale why I would use this?